odoo/addons/base/tests/test_base.py

```diff
@@ -501,6 +501,107 @@ def test_60_read_group(self):
         self.assertEqual([2, 4], [g['title_count'] for g in groups_data], 'Incorrect number of results')
         self.assertEqual([-1, 10], [g['color'] for g in groups_data], 'Incorrect aggregation of int column')
 
+    def test_61_private_read_group(self):
+        """
+        the _read_group should behave exactly like read_group (public method) except for sorting the one2many on ID
+        instead of name, so avoiding the join on the "to many" table to get the name
+        """
+        title_sir = self.env['res.partner.title'].create({'name': 'Sir...'})
+        title_lady = self.env['res.partner.title'].create({'name': 'Lady...'})
+        test_users = [
```

Comment:
The name `test_users` suggest that those are users (records).  I suggest `user_vals_list` instead.